source/chapters/user_interface.rst

```diff
@@ -591,12 +602,17 @@ control rate changes also from your computer's keyboard, see the chapter
   +0.0%, which indicates no change.
 
 **Temporary Pitch/Rate buttons (Nudge)**
-  Pushing the minus and plus buttons is like nudging the metal edge of
+  Pushing the diagonal arrow buttons is like nudging the metal edge of
   a turntable, or the outer edge of a CD player. It will give the track a push
   or pull forwards or backwards. If the buttons are released the previous tempo
   is restored. The buttons can act as either a fixed :term:`pitch bend` or a
   :term:`ramping pitch bend`.
 
+**Pitch/Rate buttons**
+  The minus button will set the pitch/rate lower while the plus button will set the pitch/rate higher.
+  These buttons, unlike the temporary pitch/rate buttons do not need to be held down. Clicking
+  on them sets the pitch/rate higher or lower for as long as the track keeps playing.
```

Comment:
the change persists even after the track is ejected (unless configured otherwise in the preferences).
```suggestion
  Clicking the :guilabel:`-` button will set the pitch/rate lower while the :guilabel:`+` button will set the pitch/rate higher. This is like moving the pitch slider up or down by a fixed amount. Unlike using the temporary pitch/rate buttons, the change is permanent so these button s don't need to be held down.
```